blueflood-core/src/main/java/com/rackspacecloud/blueflood/io/astyanax/ALocatorIO.java

```diff
@@ -46,12 +46,17 @@
      */
     @Override
     public void insertLocator(Locator locator) throws IOException {
+        Timer.Context timer = Instrumentation.getWriteTimerContext(CassandraModel.CF_METRICS_LOCATOR_NAME);
         try {
             MutationBatch mutationBatch = AstyanaxIO.getKeyspace().prepareMutationBatch();
             AstyanaxWriter.getInstance().insertLocator(locator, mutationBatch);
             mutationBatch.execute();
         } catch (Exception e) {
             throw new IOException(e);
+        } finally {
+            if ( timer != null ) {
```

Comment:
Oh, I moved the code that instantiates the timer. I don't need it anymore. I will remove.